From 5bb2356bf8c7fdf143930fc60014df83a5275a8b Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@lsi.com>
Date: Fri, 16 Aug 2013 12:47:23 -0500
Subject: [PATCH 080/158] drivers/tty: Hard code the baud rate divisors for
 now.

This assumes that the AXM55xx_533 parameter file is used.

Signed-off-by: John Jacques <john.jacques@lsi.com>
---
 drivers/tty/serial/amba-pl011.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/tty/serial/amba-pl011.c b/drivers/tty/serial/amba-pl011.c
index 5dfe6db..7071614 100644
--- a/drivers/tty/serial/amba-pl011.c
+++ b/drivers/tty/serial/amba-pl011.c
@@ -1801,8 +1801,8 @@ pl011_set_termios(struct uart_port *port, struct ktermios *termios,
 			quot -= 2;
 	}
 	/* Set baud rate */
-	writew(quot & 0x3f, port->membase + UART011_FBRD);
-	writew(quot >> 6, port->membase + UART011_IBRD);
+	writew(0x3, port->membase + UART011_FBRD);
+	writew(0x364, port->membase + UART011_IBRD);
 
 	/*
 	 * ----------v----------v----------v----------v-----
-- 
1.8.1.2

